docs/csharp/index.md

```diff
@@ -14,49 +14,22 @@ ms.assetid: 52db8280-0e53-40cf-858b-e8eef3997dea
-    * [What's new in C# 6](csharp-6.md)
-<!-- This page, or its parent should point to the Roslyn repo for folks that want
-to know what's next -->
-* [🔧 C# Concepts](concepts.md)
-    * [🔧 C# Type system](type-system.md)
-    * [Properties](properties.md)
-    * [Indexers](indexers.md)
-    * [🔧 Generics](generics.md)
-    * [Iterators](iterators.md)
-    * [🔧 Language Integrated Query (LINQ)](linq.md)
-    * [Delegates & events](delegates-events.md)
-        * [Introduction to Delegates](delegates-overview.md)
-        * [System.Delegate and the delegate keyword](delegate-class.md)
-        * [Strongly Typed Delegates](delegates-strongly-typed.md)
-        * [Common Patterns for Delegates](delegates-patterns.md)
-        * [Introduction to Events](events-overview.md)
-        * [The .NET Event Pattern](event-pattern.md)
-        * [The Updated .NET Event Pattern](modern-events.md)
-        * [Distinguishing Delegates and Events](distinguish-delegates-events.md)
-    * [🔧 Parallel programming](parallel.md)
-    * [Asynchronous programming](async.md)
-    * [🔧 Lambda Expressions](lambda-expressions.md)
-<!-- This is a sidebar the delegates topics. I don't think it
-    needs to be linked into the TOC, but I wanted to leave it 
-    to get your thoughts. If it does belong in the TOC,
-    this is the location:
-        * [Implicitly Typed Lambda Expressions](implicitly-typed-lambda-expressions.md)
--->
-    * [Expression Trees](expression-trees.md)
-        * [Expression Trees Explained](expression-trees-explained.md)
-        * [Framework Types Supporting Expression Trees](expression-classes.md)
-        * [Executing Expressions](expression-trees-execution.md)
-        * [Interpreting Expressions](expression-trees-interpreting.md)
-        * [Building Expressions](expression-trees-building.md)
-        * [Translating Expressions](expression-trees-translating.md)
-        * [Summary](expression-trees-summary.md)
-    * [🔧 Native interoperability](interop.md)
-    * [🔧 Reflection & code generation](reflection.md)
-    * [🔧 Documenting your code](codedoc.md) 
-* [🔧 Syntax Reference](syntax.md)
+The C# guide provides a wealth of information about the C# language. This site has many different audiences. Depending on your experience with programming, or with the C# language and .NET, you may wish to explore different sections of this guide.
+
+## For new developers
+
+Start with our [tutorials](tutorials/index.md) section. These tutorials show you how to create C# programs from scratch. The tutorials provide a step-by-step process to create programs. You'll learn the language concepts, and how to build C# programs on your own.
+
+If you prefer reading overview information first, try our [tour of the C# language](tour-of-csharp/index.md). It explains the concepts of the C# language. After reading this, you'll have a basic understanding of the language, and be ready to try the tutorials, or build something on your own.
+
+## For Developers new to C#
```

Comment:
nit: also on new topics we should try to follow our guidelines for sentence-case on headings and titles. less things to fix later on :smile: